src/memory/mod.rs

```diff
@@ -10,11 +10,11 @@ use core::{
 };
 use parity_wasm::elements::ResizableLimits;
 
-#[cfg(feature = "virtual_memory")]
+#[cfg(all(feature = "virtual_memory", target_pointer_width = "64"))]
```

Comment:
Maybe you can make use of https://crates.io/crates/efg here?

Reply:
i don't think it is worth it to put a whole new dependency in here for that tbh. I was also thinking about using `cfg_if` but same reason.

Reply:
besides that: let's keep the PR small and focused and do stuff like this in follow-ups if we deem it useful.